spec/javascript-spec.coffee

```diff
@@ -53,6 +53,28 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('with')
       expect(tokens[0]).toEqual value: 'with', scopes: ['source.js', 'keyword.control.js']
 
+    map =
```

Comment:
> only its test will fail and not the entire block

I think you've actually convinced me to start writing new tests this way :smile:!